src/main/scala/me/shadaj/romeojuliet/Tragedy.scala

```diff
@@ -94,15 +146,7 @@ class Juliet(friar: ActorRef) extends Actor {
           |Romeo, Romeo, Romeo! Here’s drink. I drink to thee.
           |""".stripMargin)
       context.become(sleeping)
-      context.system.scheduler.scheduleOnce(10 second) {
-        context.become(loving(lover))
```

Comment:
Never use this variant of `schedule(Once)` from within an actor, because it does stuff outside of the actor. Always schedule a message send to yourself (see below) and do the stuff that needs to be done from within the actor.